src/vivarium_public_health/population/base_population.py

```diff
@@ -202,6 +202,31 @@ def _validate_config_for_deprecated_keys(self) -> None:
                 f"Public Health. Use the new key '{mapper[key]}' instead."
             )
 
+    def _load_population_structure(self, builder: Builder) -> pd.DataFrame:
+        return load_population_structure(builder)
+
+
+class ScaledPopulation(BasePopulation):
+    """Component for scaling the population structure based on a scaling factor."""
+
+    def __init__(self, scaling_factor: str | float | pd.DataFrame):
+        super().__init__()
+        self.scaling_factor = scaling_factor
```

Comment:
Should probably add docstring for scaling_factor and specifically call out that it is multiplicative

Should it be a property?

Reply:
I don't think it needs to be a property, but I agree that the docs should make it clear that it is a a multiplicative scaling factor.